plugins/proxmox/proxmox_count

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+#%# family=auto
+#%# capabilities=autoconf
+
+# Michael Grote
+# michael.grote ät posteo.de
+# Outputs the running, stopped and total counts of ProxMox VMs and LXCs.
+# needs to be run as user: root AND group: root
+# [proxmox_count]
+# user root
+# group root
+
+if ! command -v qm &> /dev/null
+then
```

Comment:
Please emit an error message and exit with a non-zero exit code, if you encounter problematic circumstances.

(`autoconf` checks would be the proper way to do it, but you may feel free to keep it as simple as it is now)

Reply:
changed